src/plugins/data_views/public/types.ts

```diff
@@ -26,13 +77,14 @@ export interface DataViewsPublicStartDependencies {
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface DataViewsPublicPluginSetup {}
 
-export interface DataViewsServicePublic extends DataViewsService {
+export interface DataViewsServicePublic extends PublicMethodsOf<DataViewsService> {
```

Comment:
based on https://github.com/elastic/kibana/issues/126833 I'd avoid usage of `PublicMethodsOf`

Reply:
done